CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 
 (Add entries here in the order of: server, console, cli, docs, others)
 
+- server: cache default variables for non-optional variables better (fix #3773)
```

Comment:
Maybe a more user facing sentence like this?

> fixes an issue with query plan caching where valid queries would intermittently fail

Reply:
Yes, that is more helpful!